### PR TITLE
feat: style login redirect page - [ch1400]

### PR DESCRIPTION
**Pull request by kvnlnt**
Adds basic styling to login redirect page.

https://user-images.githubusercontent.com/577412/105105584-3445fd00-5a69-11eb-9d73-69eadc63b2e1.mov

@vapurrmaid who else might I add as a reviewer?

### CODE REVIEW

**Comment by vapurrmaid** [comment]
@kvnlnt We can ping @cmoog as one of the core coder-cli reviewers

**Review by rachelchhay — APPROVED** [review]
LOVE the animation you added! I think it would look nice if the green banner was a circle instead. So kinda like a big start button. But I'll leave the main design feedback to Alex.

**Review by cmoog — APPROVED** [review]
LGTM. Running `./ci/steps/fmt.sh` and `./ci/steps/lint.sh` should resolve the CI issues.

File: internal/loginsrv/server.go [hunk 1]
```diff
@@ -24,7 +24,155 @@ func (srv *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	case <-ctx.Done():
 		// Client disconnect. Nothing to do.
 	case srv.TokenChan <- token:
+		htmlDoc := `<html>
```
Comment (cmoog) [inline internal/loginsrv/server.go:27]: nit: would probably be better to define this as a constant outside of the select control-flow

**Comment by kvnlnt** [comment]
> LOVE the animation you added! I think it would look nice if the green banner was a circle instead. So kinda like a big start button. But I'll leave the main design feedback to Alex.

@alexgilev let me know what you think, feel free to blow it up.

**Comment by kvnlnt** [comment]
@cmoog moved the template outside the control flow, also put it into a parameterized function after noticing another state needing to be expressed not specified in the ticket (a session_token missing). 

@rachelchhay put in your design suggestion (made it look like a power button) as well as lifted colors from our 2.0 scheme. Also notice the "off" state I added needed for a "session_token" missing state.

https://user-images.githubusercontent.com/577412/105209058-3e660b00-5afe-11eb-8eeb-088f97ff0e06.mov

https://user-images.githubusercontent.com/577412/105209011-327a4900-5afe-11eb-8f7f-92d3a52db501.mov

File: internal/loginsrv/server.go [hunk 2]
```diff
@@ -1,30 +1,199 @@
...
+}
+
 func (srv *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	ctx := req.Context()
 
 	token := req.URL.Query().Get("session_token")
 	if token == "" {
 		w.WriteHeader(http.StatusBadRequest)
-		_, _ = fmt.Fprintf(w, "No session_token found.\n") // Best effort.
+		// _, _ = fmt.Fprintf(w, "No session_token found.\n") // Best effort.
```
Comment (cmoog) [inline internal/loginsrv/server.go:186]:
```suggestion
```

**Review by rachelchhay — APPROVED** [review]
That's exactly what I had in mind. Looks great to me! Btw, Alex is on PTO today. I'm fine with merging as is and making changes after he has a chance to take a look.

**Review by cmoog — APPROVED** [review]

**Comment by vapurrmaid** [comment]
@kvnlnt @rachelchhay @cmoog This is awesome stuff

File: internal/loginsrv/server.go (hunk 2 above)
Comment (kvnlnt) [reply internal/loginsrv/server.go]: fixed

**Pull request merged**